tests/bluetooth/bsim_bt/bsim_test_mesh/tests_scripts/provision/pb_adv_oob.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Copyright 2021 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Test provisioning with OOB authentication and with inband public key
+RunTest mesh_provision_pb_adv_oob_auth \
+	prov_device_pb_adv_oob_auth prov_provisioner_pb_adv_oob_auth
+
+# Test provisioning with OOB authentication and with OOB public key
+RunTest mesh_provision_pb_adv_oob_public_key \
+	prov_device_pb_adv_oob_public_key prov_provisioner_pb_adv_oob_public_key
+
+# Test provisioning with OOB authentication, with device OOB public key
+# but provisioner doesn't have OOB public key
+RunTest mesh_provision_pb_adv_device_w_oob_pk_prvnr_wt_pk \
+	prov_device_pb_adv_oob_public_key prov_provisioner_pb_adv_oob_auth_no_oob_public_key
```

Comment:
Unless the tests depend on each other, we generally try to have one `RunTest` entry per file, so that a failure in one of them can't mark additional failures in subsequent tests. As far as I can tell, these tests can be executed in parallel.